### ISSUE TITLE
Release for PostgreSQL 10

### ISSUE THREAD

**Issue by stub42** [wal-e/wal-e#365]
It would be great to get a release containing the PostgreSQL 10 support that landed in https://github.com/wal-e/wal-e/pull/339

**Comment by fdr** [comment]
Oh, yes, that would be good. I will cut a release soon.

**Comment by thorsten-plause** [comment]
Is there an update on the release schedule? Would be great to have the the backup functionality working in Postgres10. Thanks guys!

**Comment by technion** [comment]
@thorsten-plause  Despite the fact this issue is open, 1.1.0 is tagged and appears to contain this fix.

**Comment by thorsten-plause** [comment]
Correct, but I'd rather wait until all the tests are fixed and there is a stable release available.

**Comment by technion** [comment]
1.1.0 is a stable release?
